clang-tools-extra/clangd/refactor/Rename.cpp

```diff
@@ -811,7 +811,20 @@ llvm::Expected<RenameResult> rename(const RenameInputs &RInputs) {
   if (!Name)
     return makeError(ReasonToReject::UnsupportedSymbol);
   SymbolName OldSymbolName(Name);
-  SymbolName NewSymbolName(RInputs.NewName, AST.getLangOpts());
+  SymbolName NewSymbolName(ArrayRef<StringRef>{});
```

Comment:
Can't just do `{}`? Wouldn't hurt to have a no-arg constructor IMO

Reply:
Added a no-arg constructor.